extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSources.java

```diff
@@ -140,6 +143,19 @@ public AgroalDataSource doCreateDataSource(String dataSourceName) {
         // we first make sure that all available JDBC drivers are loaded in the current TCCL
         loadDriversInTCCL();
 
+        if (dataSourceJdbcBuildTimeConfig.tracing) {
+            TracingDriver.load();
```

Comment:
Is this really needed? Because I don't recall us requiring it in our documentation? And I suppose that forces us to add the opentracing-jdbc dependency here.

The rationale is that opentracing-jdbc should be available only if the user itself adds the dependency and use the configuration properties to enable it.